packages/apidom-ls/package.json

```
@@ -20,9 +20,61 @@
   "unpkg": "./dist/apidom-ls.browser.min.js",
   "main": "./cjs/index.cjs",
   "exports": {
-    "types": "./types/dist.d.ts",
-    "import": "./es/index.js",
-    "require": "./cjs/index.cjs"
+    ".": {
+      "types": "./types/dist.d.ts",
+      "import": "./es/index.js",
+      "require": "./cjs/index.cjs"
+    },
+    "./services/validation/providers/json-schema-validation-provider": {
```

Would be much better to make this export as concise as it can be. 

```suggestion
    "./services/validation/providers/json-schema": {
```